lib/table_of_contents/parser.rb

```diff
@@ -57,6 +57,7 @@ def parse_content
             uniq: uniq,
             text: text,
             node_name: node.name,
+            no_toc: node.attribute('class') && node.attribute('class').value.include?('no_toc'),
```

Comment:
I guess using hyphen is more common for css classname?

Reply:
Agreed, but the [name used by Kramdown is `no_toc`][toc]. Maybe Kramdown folks chose that name to avoid possible clashes with user CSS, or maybe it is just historical. Using the same class name will make it easy to switch between jekyll-toc and Kramdown tocs.

[toc]: https://kramdown.gettalong.org/converter/html.html#toc

Reply:
Good to know, thanks 👍